.mvn/wrapper/maven-wrapper.properties

```diff
@@ -6,13 +6,12 @@
 # "License"); you may not use this file except in compliance
 # with the License.  You may obtain a copy of the License at
 #
-#   https://www.apache.org/licenses/LICENSE-2.0
+#   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Fwiw, the http link redirects to https...

Reply:
yes, but this is licence, "verbatim copied":
https://www.apache.org/licenses/LICENSE-2.0

In license it does have HTTP and not HTTPS